charon/constants.py

```diff
@@ -36,12 +36,14 @@
 BASE_STATUS=['PASSED', 'FAILED']
 ANALYSIS_STATUS=['TO_ANALYZE', 'UNDER_ANALYSIS', 'ANALYZED','INCOMPLETE', 'FAILED']
 EXTENDED_STATUS=['NOT_RUNNING', 'RUNNING', 'DONE', 'FAILED']
-GENO_STATUS=['AVAILABLE', 'NOT AVAILABLE', 'PASSED', 'FAILED']
-PROJECT_STATUS=['OPEN', 'CLOSED', 'ABORTED']
-SEQUENCING_STATUS=['NEW', 'SEQUENCED', 'IGNORED']
 
 SEQ_FACILITIES=['NGI-S', 'NGI-U']
 SAMPLE_TYPES=['NORMAL', 'CANCER']
 
+GENO_STATUS={'AVAILABLE':'AVAILABLE', 'NOT AVAILABLE':'NOT AVAILABLE', 'DONE':'PASSED', 'FAILED':'FAILED'}
```

Comment:
Can we add `UNDER_ANALYSIS` to this? It would make job tracking more consistent. If you all decided not to I can just do it locally I suppose. Just let me know. @vezzi @Galithil